packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -362,7 +393,13 @@ const TrackListItemComponent = (props: TrackListItemComponentProps) => {
               <UserBadges user={user} badgeSize={12} hideName />
             </Text>
           </View>
-          {trackItemAction === 'save' ? (
+          {!isDeleted && isLocked ? (
+            <View style={styles.locked}>
+              <IconLock fill={white} width={10} height={10} />
+              <Text style={styles.lockedText}>{messages.locked}</Text>
+            </View>
+          ) : null}
+          {(isDeleted || !isLocked) && trackItemAction === 'save' ? (
```

Comment:
Super nit: Can possibly chain this with previous ternary? Idk though might be confusing. could also extract to above return statement?